evap/evaluation/models.py

```diff
@@ -828,6 +849,8 @@ def publish(self):
         self._participant_count = self.num_participants
 
         if not self.can_publish_text_results:
+            if self.voters.count() > 1:
+                logger.exception("If there is more than one voter, can_publish_text_results should be true.")
```

Comment:
If we ever hit this, it would probably be nice if `self.pk` and probably the name (in case there isn't a pk yet) were included in the log message. Maybe you can just dump `self` in there.